internal/services/bot/bot_channel_web_chat_resource_test.go

```diff
@@ -115,7 +115,10 @@ resource "azurerm_bot_channel_web_chat" "test" {
   bot_name            = azurerm_bot_channels_registration.test.name
   location            = azurerm_bot_channels_registration.test.location
   resource_group_name = azurerm_resource_group.test.name
-  site_names          = ["TestSite"]
+
+  site {
```

Comment:
We can't completely remove the deprecated property `site_named` from all tests, otherwise we're not checking that it's still usable, which it is. Please add it back to one of the tests so we can verify that it still works.

Reply:
Updated